conf/aws/application.conf

```diff
@@ -0,0 +1,162 @@
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+## Reference configuration properties for the Gobblin AWS cluster launcher
+
+#########################################################################
+## Must change the config properties below, rest can be left as-is     ##
+#########################################################################
+
+# Cluster name
+gobblin.aws.cluster.name=GobblinAWS
+
+# Zookeeper connection string
+# Note: You can use the example below, but the example Zookeeper may or may not be available
+# Example: gobblin.cluster.zk.connection.string="ec2-54-149-223-115.us-west-2.compute.amazonaws.com:2181"
+gobblin.cluster.zk.connection.string="<Zookeeper connection string>"
+
+# AWS Credentials
+gobblin.aws.service.access="<AWS Access Key>"
+gobblin.aws.service.secret="<AWS Secret Key>"
+
+# Leave as-is for sample wikipedia pull job
+# -- OR --
+# Specify S3 URI for job configuration:
+# Steps to obtain this URI:
+# 1. Create a S3 folder in a bucket, say the bucket name is "s3bucket" and folder is "/gobblin-confs/job-conf/"
+# 2. Zip your pull files in a zip and upload to this folder. Say your zip file name is myPullFiles.zip
+# 3. Specify S3 URI for this zip file in the config below
+#    For the description above: "https://s3-region.amazonaws.com/s3bucket/gobblin-confs/job-conf/myPullFiles.zip"
+gobblin.aws.job.conf.s3.uri="https://s3-us-west-2.amazonaws.com/gobblin-libs/job-config/wikipedia.zip"
+
+# Specify S3 URI for cluster configuration
+# Steps to obtain this URI:
+# 1. Create a S3 folder in a bucket, say the bucket name is "s3bucket" and folder is "/gobblin-confs/cluster-conf/"
+# 2. After editing this application.conf: upload this application.conf, log4j-aws.properties and quartz.properties to
+#    /s3bucket/gobblin-confs/cluster-conf/
+#    Note: Make sure to replace AWS credential values with dummy values.
+#          We don't read AWS credentials from application.conf in S3
+# 3. Specify S3 URI for this folder in the config below
+#    For the description above: "https://s3-region.amazonaws.com/s3bucket/gobblin-confs/cluster-conf/"
+# Example: gobblin.aws.master.s3.conf.uri="https://s3-us-west-2.amazonaws.com/gobblin-libs/cluster-config/"
+gobblin.aws.master.s3.conf.uri="https://s3-region.amazonaws.com/s3bucket/gobblin-confs/cluster-conf/"
+
+#########################################################################
+## Advanced config properties. You need not change anything below      ##
+#########################################################################
+
+## Gobblin version
+# Note:
+# 1. If you change Gobblin version here, make sure that those version jars are
+# .. on S3 url referred by: gobblin.aws.master.s3.jars.uri
+# 2. Version below is available on the S3 at: https://s3-us-west-2.amazonaws.com/gobblin-libs/latest-jars/
```

Comment:
is this link suppose to be publicaly accessible? i can't seem to find it

will this go away when the gobblin ami is built?